src/features/emails/components/EmailKPIChart.tsx

```diff
@@ -6,31 +6,70 @@ import { ZetkinEmail } from 'utils/types/zetkin';
 
 type Props = {
   email: ZetkinEmail;
+  secondaryEmail?: ZetkinEmail | null;
+  secondaryTotal?: number | null;
+  secondaryValue?: number | null;
   title: string;
   total: number;
   value: number;
 };
 
-const EmailKPIChart: FC<Props> = ({ email, title, total, value }) => {
+const EmailKPIChart: FC<Props> = ({
+  email,
+  secondaryEmail,
+  secondaryTotal,
+  secondaryValue,
+  title,
+  total,
+  value,
+}) => {
   const theme = useTheme();
 
   const data = [
     {
       data: [
         {
           x: 'Opened',
-          y: value,
+          y: value / total,
         },
         {
           x: 'Opened 2',
           y: 0,
         },
+        {
+          x: 'Other',
+          y: 1 - value / total,
+        },
```

Comment:
The `x` is the name of the radial bar, which I never intended to show so what is used here is purely for internal (code) use.

Reply:
You have linked the same commit twice. I believe the second commit that you intended to reference is 2fdd0dd48d635e05cd21bda72aa9bb62dcf146e9.

I will try to explain this from memory without going back and experimenting and hopefully I remember my reasoning correctly.

The nivo radial bar allows for multiple "series", where each series will be rendered as a separate circle (radial bar). Each circle (radial bar) can have several segments, similarly to how a normal bar chart can have "stacked" bars. If the segments do not total 100% the remainder of the circle will be filled with the `tracksColor`.

Our use case is not 100% what radial bars were intended for, for two reasons:

1. Because we wish to render what is essentially the same data point ("open rate") in both circles, but in different colors. Normally, values for the same data point would have the same color across all series.
2. Because we wish to render a percentage, so a single value of 5% should not fill the entire circle, even if it's the only value (and thus constitutes 100% of the maximum)

These didn't cause big problems when I had just one series (before the comparison implemented in 2fdd0dd48d635e05cd21bda72aa9bb62dcf146e9) because 1 is only relevant for multiple series and 2 could be solved using `maxValue`, but when I have two emails the `maxValue` is actually different for each email and can no longer be used.

So what I've ended up doing is having one data point for the open rate of the primary email (which will be zero for the secondary email) and one data point for the open rate of the secondary email (which will be zero for the primary), and then finally one data point for the unopened.